images/sdn/Dockerfile.rhel8

```diff
@@ -0,0 +1,25 @@
+# This image contains the RHEL8 base and ovs/ovn fastdatapath RHEL8 rpms needed to set up ovs/ovn
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+RUN INSTALL_PKGS=" \
+      which tar wget hostname shadow-utils \
+      socat findutils lsof bind-utils gzip \
```

Comment:
these could be pulled out into a rhel8 base image that parallels the rhel7 base image we use for most other things.